src/contract-load.ts

```diff
@@ -63,13 +63,12 @@ export function createContractExecutionEnvironment(
   contractSrc: string,
   contractId: string,
   contractOwner: string,
+  customReadContract?: any,
 ) {
   const returningSrc = normalizeContractSource(contractSrc);
-  const swGlobal = new SmartWeaveGlobal(arweave, { id: contractId, owner: contractOwner });
+  const swGlobal = new SmartWeaveGlobal(arweave, { id: contractId, owner: contractOwner, customReadContract });
```

Comment:
from my understanding - after this change different "readContract" functions might be used inside SWC code and from the SDK itself (SDK always uses built-in readContract version). Not sure if that's a good solution - shouldn't this be consistent - ie. both SDK and version exposed within SWC must the same?

Reply:
![image](https://user-images.githubusercontent.com/6033345/124305392-e6561a80-db19-11eb-91a1-f03f3c184b42.png)

Reply:
We already have our own custom `readContract` (we do not use the SDK one), to be consistent we have to also make it so contract execution use that new one too